### DISCUSSION TITLE
Failed to initialize submodules (stylesheet error)

### DISCUSSION THREAD

**Discussion by yinanazhou** [music-encoding/music-encoding#814] [Q&A] [answered]
Hi, I was following the instructions on Building MEI in README. But after I ran the command line `git submodule init`, I got an error saying:
```
': not a valid identifier/bin/gettext.sh: line 88: export: `dashless
': not a valid identifier/bin/gettext.sh: line 88: export: `USAGE
Submodule 'Stylesheets' (https://github.com/TEIC/Stylesheets) registered for path 'submodules/Stylesheets'
```
I'm using windows10, and I have also tried to use Oxygen. It also reported an error about stylesheet:
```
Transformation failed. Fatal error during transformation using ***\Oxygen XML Editor 21\frameworks\tei\xml\tei\stylesheet\odds\odd2odd.xsl: Relative URI 
 {***\...} is invalid: Illegal character in opaque part at index 2: 
***\music-encoding\customizations/***\music-encoding\source\mei-source.xml;
 SystemID: file:/***/Oxygen%20XML%20Editor%2021/frameworks/tei/xml/tei/stylesheet/odds/odd2odd.xsl;
Line#: 250; Column#: 25
```

**Comment by ahankinson** [comment]
I’ve moved this to discussions as it is a help request.

**Comment by ahankinson** [comment]
What is the full path to your stylesheets? A first guess is that you have a bad character in your path.

**Reply by yinanazhou** [reply]
Hi, the full path is:    
F:\McGill\SIMSSA\oxygen\22.1\Oxygen XML Editor 21\frameworks\tei\xml\tei\stylesheet\odds\odd2odd.xsl

**Comment by ahankinson** [comment]
https://github.com/msys2/MSYS2-packages/issues/735

looks like this thread should help

**Comment by ahankinson** [comment]
@yinanazhou if the issue I linked above helped you out, please mark this discussion as 'answered'. Thank you!

**Comment by yinanazhou** [comment]
Hi, my workaround is to use WSL instead. Thank you for the help. I'll mark it as answered.

**Comment by yinanazhou** [comment, marked as answer]
Hi, we contacted support from help. It's a path thing. If we "copy location" in the oxygen tab and paste it in oxygen, the path starts with "file:/". This header is essential. Thank you for your help. I hope this can help other people.
